src/templates/app/cashier/payment_methods.jsx

```diff
@@ -25,30 +25,9 @@ const TableValues = ({ value }) => {
     );
 };
 
-const ReferenceLink = ({ href, className = '', title = '' }) => (
-    <a
-        className={`payment-methods__reference ${className}`}
-        href={href}
-        target='_blank'
-        aria-disabled={!href}
-        title={title}
-        rel='noopener noreferrer'
-    />
-);
-
-const ReferenceLinks = ({ pdf_file, video_link }) => (
+const ReferenceLinks = () => (
```

Comment:
Do we still need this empty method? Isn't it better to remove it if it's unnecessary?

Reply:
And as I'm seeing the card, it asks for removing video links only, do we have any extra request for removing `PDF` as well?

Reply:
Yes, PDF should be removed as well, it's just not mentioned in the card

Reply:
About the method: I think we still need it so it would be really easy and fast to add new links to updated videos and PDFs, when they will be available. The main reason for deleting them was that all of them are out of date.

Reply:
Alright, so isn't it better to just remove the props passed to the <ReferenceLinks /> (in the usages) and let these methods stand steal for future use?

Reply:
done